src/sentry/models/group.py

```diff
@@ -377,6 +377,39 @@ def get_latest_event(self):
                 self._latest_event = None
         return self._latest_event
 
+    def get_latest_event_for_environments(self, environments=[]):
```

Comment:
`get_latest_event_for_environments` and `get_oldest_event_for_environments` are basically the same thing, could you DRY them up a bit?